box-share-sdk/src/main/java/com/box/androidsdk/share/vm/InviteCollaboratorsDataWrapper.java

```diff
@@ -0,0 +1,39 @@
+package com.box.androidsdk.share.vm;
+
+/**
+ * A special extension of DataWrapper to hold data for inviting a new Collaborator
+ */
+public class InviteCollaboratorsDataWrapper extends DataWrapper<String> {
+
+    private boolean mInvitationFailed;
+
+    public InviteCollaboratorsDataWrapper(String data, int strCode, boolean invitationFailed) {
+        this.mData = data;
+        this.mStrCode = strCode;
+        this.mInvitationFailed = invitationFailed;
+    }
+
+    /**
+     * Returns true if all invitations succeeded.
+     * @return true if all invitations succeeded
+     */
+    public boolean isSuccess() {
```

Comment:
@Override annotation would be helpful here

Reply:
Got it. I will add the annotation.